harness/determined/cli/user.py

```diff
@@ -170,7 +170,13 @@ def whoami(parsed_args: Namespace) -> None:
 
 args_description = [
     Cmd("u|ser", None, "manage users", [
-        Cmd("list ls", list_users, "list users", [], is_default=True),
+        Cmd("list ls", list_users, "list users", [
+            Arg(
+                "--all",
+                "-a",
+                action="store_true",
+                help="log out of all cached sessions for the current master",
```

Comment:
.

Reply:
comment string is wrong?

Reply:
also, same comment about the naming. if we're introducing a filter on active status, can we call it something more explicit like "active"?

Reply:
yep copypasta fixed